enclave-runtime/src/sidechain_impl.rs

```diff
@@ -3,17 +3,19 @@
 //! Todo: Once we have put the `top_pool` stuff in an entirely different crate we can
 //! move most parts here to the sidechain crate.
 
-use crate::{execute_top_pool_trusted_calls, prepare_and_send_xts, Result as EnclaveResult};
+use crate::{execute_top_pool_trusted_calls, Result as EnclaveResult};
```

Comment:
The main reason we're doing this refactoring: refactor and remove the `prepare_and_send_xts` function from the `enclave-runtime`, so we can move this `sidechain_impl` code back into the sidechain crate.

(We still have the `execute_top_pool_trusted_calls` function in `enclave-runtime` - that will be refactored in the next PR)